src/Components/CookieBanner.js

```diff
@@ -1,13 +1,20 @@
 import * as React from "react";
 import { Modal } from "react-bootstrap";
+
 import { useCookieConsent } from "./CookieConsentContext";
 import Header from "./Header";
 import Body from "./Body";
 import Footer from "./Footer";
 import "../assets/main.scss";
 
 export default function CookieBanner() {
-  const { bannerVisibility, switchBannerMode, I18n } = useCookieConsent();
+  const { bannerVisibility, switchBannerMode, I18n, config } =
+    useCookieConsent();
```

Comment:
Why would you:
1) you grab config
`const { bannerVisibility, switchBannerMode, I18n, config } =
    useCookieConsent();`
 
2) Pass config into `<Body config={config} />`

3) and than in Body:
```
export default function Body({ config }) {
  const { isExtendedMode, I18n } = useCookieConsent();
```
?
  
  It's confusing. Why don't we pass all params (or something)?